package.json

```
@@ -102,7 +102,8 @@
     "socks-proxy-agent": "^5.0.0",
     "update-notifier": "4.1.0",
     "uuid": "3.4.0",
-    "xml2js": "^0.4.23"
+    "xml2js": "^0.4.23",
+    "keytar": "5.4.0"
```

```suggestion
    "xml2js": "^0.4.23"
```

This change seems unrelated (unless I've missed something)

---

Yeah I'm not sure why that's in there tbh, I'll check it out.